### PR TITLE
added Publish and Explore components

### PR DESCRIPTION
**Pull request by bones7242**
Just getting familiar with how Vue works.  I recreated the publish and explore pieces of the original spee.ch as components in our Nuxt project.

### CODE REVIEW

File: src/components/Explore.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="explore-wrapper">
+    <h2>Explore</h2>
+    <div class="explore-body">
+      <ul id="explore-list">
+        <li v-for="item in items">
+          <a v-bind:href="item.msg">{{item.msg}}</a>
```
Comment (Rob-pw) [inline src/components/Explore.vue]: https://vuejs.org/v2/api/#v-bind

Shorthand form would be `:href="item.msg"`, it's a tad easier to type ^^.

File: src/components/Explore.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data: () => {
```
Comment (Rob-pw) [inline src/components/Explore.vue]: Can take the form 

`
data: () => ({
  property: value
})
`

File: src/components/Publish.vue [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  data: () => {
+    return {
+      showClaimInput: false
+    }
+  }
+}
+</script>
+
+<style>
+  .publish-wrapper{
```
Comment (Rob-pw) [inline src/components/Publish.vue]: (low priority, don't really care) some lowly spaces for consistency :p

**Review by Rob-pw — CHANGES_REQUESTED** [review]

**Comment by Rob-pw** [comment]
Just a couple of small tweaks, but this is fantastic, excellent job!

File: src/components/Publish.vue (hunk 3 above)
Comment (bones7242) [reply src/components/Publish.vue]: wasn't quite sure which spacing you preferred, but I will update to match the style tag in index.vue

**Comment by bones7242** [comment]
I made requested changes and rebased the commit :)

**Pull request merged**